man/pkcs11sign.cnf.5

```diff
@@ -82,6 +77,22 @@ default provider as forward.
 The syntax for this parameter is "provider=<name_of_forward_provider>". See
 the configuration example for more details.
 .PP
+.TP
+.BR pkcs11sign-module-init-args " (optional, not PKCS#11-3.0 conform)"
+The pkcs11sign-module-init-args takes a parameter string whose
+reference is passed to the Cryptoki module as
+.IR pReserved " in"
+.IR CK_C_INITIALIZE_ARGS
+during initialization.
+.IP
+Note: The PKCS#11 standard v3.0 specifies that the initialization of a
+Cryptoki module fails if
+.IR pReserved
+is not a NULL_PTR. This parameter will only work with Cryptoki modules
+(e.g.  libnss) which does not implement this strict behavior. Cryptoki
+modules, which strictly implements the PKCS#11 standard v3.0 will fail
+if this parameter is set.
+.PP
```

Comment:
You mixed singular and plural here. I would recomment this wording:

This parameter will only work with Cryptoki modules (e.g.  libnss) which do not implement this strict behavior.
A Cryptoki module, which strictly implements the PKCS#11 standard will fail on C_Initialize with CKR_ARGUMENTS_BAD if this parameter is set.

Reply:
ok